worlds/faxanadu/Locations.py

```diff
@@ -0,0 +1,199 @@
+from typing import List
+
+
+class LocationType():
+    world = 1 # Just standing there in the world
+    hidden = 2 # Kill all monsters in the room to reveal, each "item room" counter tick.
+    boss_reward = 3 # Kill a boss to reveal the item
+    shop = 4 # Buy at a shop
+    give = 5 # Given by an NPC
+    spring = 6 # Activatable spring
+    boss = 7 # Entity to kill to trigger the check
+
+
+class ItemType():
+    unknown = 0 # Or don't care
+    red_potion = 1
```

Comment:
These could probably be IntEnum, but that's up to you. idk it well enough to say for certain whether that would be better.

Reply:
But also the () are not necessary if you aren't making them an IntEnum or some other type, so those can be removed if you're not making it some type.